src/ms-prefixed.js

```diff
@@ -37,7 +37,8 @@ export const getMessageContents = (message) => {
 export const addKeyToSession = (options, session, event) => {
   const playreadyOptions = options.keySystems[PLAYREADY_KEY_SYSTEM];
 
-  if (typeof playreadyOptions.getKey === 'function') {
+  if (typeof playreadyOptions === 'object' &&
```

Comment:
I don't think this check is needed. if `playreadyOptions` is a `'string'` then `typeof playreadyOptions.getKey` will return `'undefined'`

Reply:
I guess it depends on what we want to allow, and brings up an interesting question. If someone wants to specify nothing (to use event.destinationURL), then they can either leave out the playready options entirely/put null (which would require the object check here), or put false (would work with typeof false.getKey). Right now it allows anything, but maybe we should consider a warning if a user tries to use one of those methods (to protect them from just forgetting the option).